### PR TITLE
Fix dependencies and vulnerabilities

### PR DESCRIPTION
**Pull request by trevorlinton**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,18 @@
-FROM node:10-alpine
+FROM node:14-alpine
 
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
 ARG NODE_ENV
 ENV NODE_ENV $NODE_ENV
 COPY package*.json /usr/src/app/
-RUN npm install -g webpack@4.41.2 && npm install && npm cache clean --force
```
Comment (sbeck14) [inline Dockerfile:9]: Just curious, why are we splitting this line up? I thought we wanted these commands to be cached in one Docker layer.

**Review by sbeck14 — APPROVED** [review]
Approved, tests passing on my machine

File: Dockerfile (hunk 1 above)
Comment (trevorlinton) [reply Dockerfile:9]: Oh, I guess I wasn't aware we wanted to have these in one layer, what's the purpose of having these in a layer? Is the docker image size significantly larger?  I can revert this.

Comment (sbeck14) [reply Dockerfile:9]: It's not a big enough deal to revert, I think there was a minor space improvement? I can't remember exactly. It's fine to leave it though :)

**Pull request merged**